src/rgw/rgw_main.cc

```diff
@@ -130,6 +130,12 @@ struct RGWRequest
     utime_t t = ceph_clock_now(g_ceph_context) - ts;
     dout(2) << "req " << id << ":" << t << ":" << s->dialect << ":" << req_str << ":" << (op ? op->name() : "") << ":" << msg << dendl;
   }
+  
+  utime_t time_elapsed() {
+    utime_t elapsed = ceph_clock_now(g_ceph_context) - ts;
```

Comment:
yes, ts is a global variable of RGWRequest struct, which we initialize everytime when we start logging for a request through the function log()
of the same struct